pkg/machines/components/libvirtSlate.jsx

```diff
@@ -76,6 +77,9 @@ class LibvirtSlate extends React.Component {
         } else if (name && activeState === 'unknown') { // name === 'unknown' first
             message = _("Connecting to Virtualization Service");
             icon = (<div className="spinner spinner-lg" />);
+        } else if (loadingResources) {
+            message = _("Loading Libvirt Objects");
```

Comment:
I think this is too much of a technicality to get exposed.
Just 'Loading State' / 'Loading Resources' or something similar would do better I think. But I would avoid mentioning Libvirt.